ui/tests/integration/components/sync/secrets/landing-cta-test.js

```diff
@@ -20,11 +20,35 @@ module('Integration | Component | sync | Secrets::LandingCta', function (hooks)
   setupMirage(hooks);
 
   hooks.beforeEach(function () {
+    this.version = this.owner.lookup('service:version');
+    this.persona = this.owner.lookup('service:persona');
+    this.version.type = 'enterprise';
+    this.version.features = [];
+    this.version.activatedFeatures = [];
+
     this.transitionStub = sinon.stub(this.owner.lookup('service:router'), 'transitionTo');
   });
 
-  test('it should render promotional copy for community or enterprise version without feature', async function (assert) {
-    await render(hbs`<Secrets::LandingCta @isActivated={{false}} @hasSecretsSync={{false}} /> `, {
+  test('it should render promotional copy for community', async function (assert) {
+    this.version.type = 'community';
+    this.secretsSyncPersona = this.persona.secretsSyncPersona;
+    await render(hbs`<Secrets::LandingCta @secretsSyncPersona={{this.secretsSyncPersona}} /> `, {
```

Comment:
Since this is an integration test, this can simply be tested as 
```suggestion
    await render(hbs`<Secrets::LandingCta @secretsSyncPersona="SYNC_ACTIVATED" /> `
```
or whatever string you want to test, and then a persona test should be added to map the various version states to the desired persona strings

Reply:
Good point!